app/controllers/application_controller.rb

```diff
@@ -6,7 +6,8 @@ class ApplicationController < ActionController::Base
 
   def invalid_record(redirect_path:, resource:)
     respond_to do |format|
-      format.html { redirect_to redirect_path, alert: t(:invalid_name) }
+      # format.html { redirect_to redirect_path, alert: t(:invalid_name) }
+      format.html { redirect_to redirect_path, alert: resource.errors.messages }
```

Comment:
To get the following result:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/12144785/224752416-ab68ac3a-c6ab-4620-af09-df9e58dbe809.png">
You can do this:
```rb
# application_controller.rb
def invalid_record(redirect_path:, resource:)
  respond_to do |format|
    format.html { redirect_to redirect_path, alert: resource.errors.full_messages }
    format.json { render json: { errors: resource.errors }, status: :unprocessable_entity }
  end
end
```
```rb
# application.html.erb

<% if alert %>
    <div class="notification is-danger">
      <% for error in flash[:alert] %>
        <%= error %><br>
      <% end %>
    </div>
<% end %>
```

Reply:
Not great but better than what you currently have. You could also add periods at the end of error messages if you want:
```rb
resource.errors.full_message.map { |error| error + '.' }
```

Reply:
No need to do that right now but in the future you'll be able to translate those errors by doing this: https://dev.to/risafj/the-basics-of-rails-i18n-translate-errors-models-and-attributes-84d